base/deprecated.jl

```diff
@@ -539,6 +539,63 @@ end
 
 export MathConst, @math_const
 
+# 11551
+function utf16_is_surrogate(chr::UInt16)
```

Comment:
OK, good point! Thanks! (I'm just not yet good at macros, but this is probably easy enough for me to learn how quickly)